<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sysinfo</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="generaltab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Current Language:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" colspan="2">
        <widget class="QLineEdit" name="curLanguageLineEdit"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Current Country:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2" colspan="2">
        <widget class="QLabel" name="countryCodeLabel">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Available Languages:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2" colspan="2">
        <widget class="QComboBox" name="languagesComboBox"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Version</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QComboBox" name="versionComboBox">
         <item>
          <property name="text">
           <string>Os</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>QtCore</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Firmware</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="3" colspan="2">
        <widget class="QLineEdit" name="versionLineEdit"/>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Feature supported</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2" colspan="2">
        <widget class="QComboBox" name="featureComboBox">
         <item>
          <property name="text">
           <string>Bluetooth</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Camera</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>FmRadio</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Irda</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Led</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Memcard</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Usb</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Vibrate</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>WLan</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sim</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Location</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>VideoOut</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Haptics</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="4">
        <widget class="QLineEdit" name="featuresLineEdit"/>
       </item>
       <item row="5" column="1">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>291</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="sysdevicetab">
      <attribute name="title">
       <string>Device</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0" colspan="2">
        <widget class="QProgressBar" name="batteryLevelBar">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="0" column="2" rowspan="4" colspan="2">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Power state</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="0">
           <widget class="QRadioButton" name="radioButton">
            <property name="text">
             <string>Unknown Power</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="radioButton_2">
            <property name="text">
             <string>Battery Power</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="radioButton_3">
            <property name="text">
             <string>Wall Power</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QRadioButton" name="radioButton_4">
            <property name="text">
             <string>Wall Power charging Battery</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>298</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>IMEI:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="ImeiLabel">
         <property name="text">
          <string>xx</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>IMSI:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="imsiLabel">
         <property name="text">
          <string>xx</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>Manufacturer:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="manufacturerLabel">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>Model</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="modelLabel">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_22">
         <property name="text">
          <string>Product:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="productLabel">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QCheckBox" name="deviceLockCheckBox">
         <property name="text">
          <string>Device locked</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_23">
         <property name="text">
          <string>Sim status:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1" colspan="2">
        <widget class="QComboBox" name="simComboBox">
         <item>
          <property name="text">
           <string>Sim not available</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Single available</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dual available</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Locked</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="8" column="3">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>201</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_24">
         <property name="text">
          <string>current Profile:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1" colspan="2">
        <widget class="QComboBox" name="profileComboBox">
         <item>
          <property name="text">
           <string>Unknown Profile</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Silent Profile</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Normal Profile</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Loud profile</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Vib profile</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Offline profile</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Powersave profile</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Custom profile</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_25">
         <property name="text">
          <string>Input method:</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QLabel" name="inputMethodLabel">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QCheckBox" name="bluetoothOnCheckBox">
         <property name="text">
          <string>Bluetooth on</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="sysdisplaytab">
      <attribute name="title">
       <string>Display</string>
      </attribute>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>14</x>
         <y>30</y>
         <width>67</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>brightness</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="brightnessLineEdit">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>30</y>
         <width>114</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>14</x>
         <y>60</y>
         <width>72</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>color depth</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="colorDepthLineEdit">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>60</y>
         <width>114</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="sysmemorytab">
      <attribute name="title">
       <string>Storage</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QTreeWidget" name="storageTreeWidget">
         <column>
          <property name="text">
           <string>Volume Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Type of Volume</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Total Disk Space</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Available Disk Space</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="sysnetworktab">
      <attribute name="title">
       <string>Network</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QComboBox" name="netStatusComboBox">
            <item>
             <property name="text">
              <string>Gsm</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Cdma</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Wcdma</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Wlan</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Ethernet</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Bluetooth</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <spacer name="horizontalSpacer_8">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Network Status</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="cellNetworkStatusLabel">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="text">
             <string>.</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Network Signal Strength</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QProgressBar" name="signalLevelProgressBar">
            <property name="value">
             <number>0</number>
            </property>
            <property name="format">
             <string>%v</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_26">
            <property name="text">
             <string>MAC address:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="macAddressLabel">
            <property name="text">
             <string>.</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_27">
            <property name="text">
             <string>Interface Name:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QLabel" name="InterfaceLabel">
            <property name="text">
             <string>.</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Cell Id:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="cellIdLabel">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="text">
             <string>.</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Location Area Code</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="locationAreaCodeLabel">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="text">
             <string>.</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Current MMC:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="currentMMCLabel">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="text">
             <string>.</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>136</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="3">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Home MMC:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QLabel" name="homeMMCLabel">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="text">
             <string>.</string>
            </property>
           </widget>
          </item>
          <item row="2" column="5">
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>135</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Current MNC:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="currentMNCLabel">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="text">
             <string>.</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>136</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="3">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Home MNC:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="4">
           <widget class="QLabel" name="homeMNCLabel">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="text">
             <string>.</string>
            </property>
           </widget>
          </item>
          <item row="3" column="5">
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>135</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Network Name</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1" colspan="5">
           <widget class="QLabel" name="operatorNameLabel">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="text">
             <string>.</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>213</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="screenTab">
      <attribute name="title">
       <string>Screen Saver</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0" colspan="3">
        <widget class="QCheckBox" name="saverInhibitedCheckBox">
         <property name="text">
          <string>Screen Saver Inhibited</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
